crnk-client/src/main/java/io/crnk/client/internal/ClientStubBase.java

```diff
@@ -141,4 +141,12 @@ protected RuntimeException handleError(HttpAdapterResponse response) throws IOEx
 	public enum ResponseType {
 		NONE, RESOURCE, RESOURCES
 	}
+	
+	public JsonApiUrlBuilder getUrlBuilder() {
```

Comment:
I think it would already be sufficient to be able to get/set the serializer, the JsonApiUrlBuilder is rather something internal I think people do not need to customize? currently it also  sits in the internal package. The getter/setters could then forward the call to JsonApiUrlBuilder.

Reply:
you're right. deleting this PR, I'm gonna send another in a few minutes